packages/react/rollup.config.js

```diff
@@ -14,6 +15,7 @@ const config = defineConfig([
       sourcemap: false,
     },
     plugins: [
+      externals({ include: /^@aws-amplify/ }),
```

Comment:
For my knowledge, what does this do?

Reply:
Ideally that plugin will take care of all types of dependencies declared in `package.json`. However, we do have a few `@aws-amplify` phantom dependencies used in hook. They are neither imported right from the `aws-amplify` nor  being declared in dependencies. So we have to include them explicitly

Reply:
Got it, lgtm 👍